testing/tests/DevExpress.ui.widgets.scheduler/appointmentPopup.tests.js

```diff
@@ -273,7 +273,7 @@ QUnit.test("startDateBox value should be valid", function(assert) {
     var form = this.instance.getAppointmentDetailsForm(),
         startDateBox = form.getEditor("startDate");
 
-    startDateBox.option("value", null);
+    startDateBox.option("value", undefined);
```

Comment:
Use the [reset](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxDateBox/Methods/#reset) method instead of startDateBox.option("value", undefined);

Reply:
option values are pre-validating in Appointment_form, method _validateAppointmentFormDate . Method "reset" has default value "null", which is acceptable for formDate. Thus, test will not show error in some cases.